Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3
```

Comment:
Que tal usaros a distribuição `slim` (economizar espeaço em disco) e especificar a versão do Python? Qual usamos no Heroku, @anapaulagomes?

Reply:
Usamos a `3.8.0`. Não especificamos nenhuma versão do Python. No CI, pegamos a última da versão disponível no Gitlab CI e no Heroku a última suportada no Python Buildpack. Será que deveríamos fixar uma?

Reply:
Concordo em fixar a versão sim. E manualmente mudar quando tivermos versões novas. Tomando o cuidado de testar tudo. Dá pra fixar a versão do python no heroku.

PS: A versão do python que eles tem disponível é a `python-3.8.2`

Reply:
Diria que minha experiência com Django me convenceu que isso é uma _excelente_ prática, pois vex ou outra podem ter [bugs dependendo de combinações específicas ade Python e Django, mesmo em _minor releases_](https://code.djangoproject.com/ticket/31067).

Reply:
O mesmo vale para combinações ainda especificas: subdependências rodando em determinada versão de python + SO.
Quem salva a pele normalmente é o [pipenv](https://github.com/pypa/pipenv)

Reply:
Django 3.0 supports Python 3.6, 3.7, and 3.8. We highly recommend and only officially support the latest release of each series.

https://docs.djangoproject.com/en/3.0/releases/3.0/

Eu vou pegar a versão mais recente, a 3.8.2 para ver se passa. Vou procurar a `slim` também para deixar mais leve.

Reply:
Eu tenho uma dúvida aqui, o `slim` dá pra instalar o postgres?

Reply:
Sim, o `slim` é um GNU/Linux Debian, então é só usar o `apt`. O nome do pacote no `apk` e no `apt` é diferente, mas rola sim : )